src/data/set/intervals/infinite.lean

```diff
@@ -54,7 +54,7 @@ begin
 end
```

Comment:
While you're at it, could you please change the proof of `Iio.infinite` to something like `rcases no_bot b with ⟨a, ha⟩, exact (Ioo.infinite ha).mono Ioo_subset_Iio_self`?

Reply:
Well... no, because `Ioo.infinite` requires `densely_ordered`.